traefik-image/traefik-image.kiwi.ini

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- OBS-AddTag: _NAMESPACE_/traefik:%%LONG_VERSION%% _NAMESPACE_/traefik:%%LONG_VERSION%%-<RELEASE> -->
+
+<image schemaversion="6.5" name="_PRODUCT_-traefik-image">
+  <description type="system">
+    <author>SUSE Containers Team</author>
+    <contact>containers@suse.com</contact>
+    <specification>nginx ingress controller running on a _DISTRO_ container guest</specification>
+  </description>
+  <preferences>
+    <type
+      image="docker"
+      derived_from="obsrepositories:/_BASEIMAGE_">
+      <containerconfig
+        name="_NAMESPACE_/traefik"
+        tag="%%SHORT_VERSION%%"
+        maintainer="SUSE Containers Team &lt;containers@suse.com&gt;">
+        <entrypoint execute="/entrypoint.sh"/>
+        <subcommand clear="true"/>
+        <expose>
+          <port number="80"/>
+        </expose>
+        <volumes>
+          <volume name="/tmp"/>
+        </volumes>
+        <labels>
+           <label name="org.label-schema.description" value="A modern reverse-proxy"/>
```

Comment:
Are those labels mandatory? I think we should follow the https://fate.suse.com/326856 the base images just started adopting so and it is still to be seen how we want to add them in CaaSP/Kubic. If not mandatory I would just leave them without those labels and just add them for all images at once following a clear criteria.

There is a card on the backlog regarding that in [v4](https://trello.com/c/rp3dq9AV/44-add-labels-schema-to-our-caasp-images-for-sle15)

Reply:
I don't think they are needed -- I've removed it.